psd-web/app/services/create_organisation.rb

```diff
@@ -0,0 +1,25 @@
+class CreateOrganisation
```

Comment:
It took me a few mins to realise that this sets up a bunch of different models, so I wonder whether a more verbose class name would help:

```suggestion
class CreateOrganisationWithTeamAndAdminUser
```